homeassistant/components/bmw_connected_drive/manifest.json

```diff
@@ -6,5 +6,5 @@
   "documentation": "https://www.home-assistant.io/integrations/bmw_connected_drive",
   "iot_class": "cloud_polling",
   "loggers": ["bimmer_connected"],
-  "requirements": ["bimmer_connected==0.13.0"]
+  "requirements": ["bimmer_connected==0.13.2"]
```

Comment:
Please add a link to a changelog or GitHub commit compare view for the version bump in the PR description. Thanks!

Reply:
Forgot that, sorry. It is there now.